.changesets/when-a-deployment-tool-provides-the-revision-file--appsignal-will-now-use-that-to-set-the-revision-config.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: minor
```

Comment:
@shairyar Sorry I didn't catch this earlier. This shouldn't be a minor bump, it should be a patch bump. Can you amend it?